templates/infrastructure/environment/gcp/redis/terragrunt.hcl

```diff
@@ -0,0 +1,28 @@
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  # the below config is an example of what the config should like
+  source = "github.com/argonautdev/tf-modules.git//modules/gcp/memorystore-redis?ref={{.RefVersion}}"
+}
+
+dependency "{{.Spec.vpc_id}}" {
+  config_path = "../vpc_{{.Spec.vpc_id}}"
+}
+
+inputs = {
+  default_labels = {
+    "argonaut-id"        = "{{.Spec.id}}"
+    "argonaut-name"        = "{{.Spec.name}}"
+    "argonaut-type"        = "redis"
+    "argonaut-manager"     = "argonautdev"
+    "argonaut-environment" = "{{.Environment.Name}}"
+  }
+  vpc_network_name               = dependency.{{ .Spec.vpc_id }}.outputs.network_name
+  tier                   = "{{.Spec.tier}}"
+  memory_size_gb         = "{{.Spec.memory_size_gb}}"
```

Comment:
This is a user input. I do not see it in the `modules.yaml` file @shubhamgrg04

Reply:
this is not needed